diff --git a/CMakeLists.txt b/CMakeLists.txt
index 1ce4af7..00b33ba 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -296,6 +296,8 @@ endif()
 
 # For registration to libraw_config.h
 MACRO_BOOL_TO_01(RAWSPEED_SUPPORT_CAN_BE_COMPILED LIBRAW_USE_RAWSPEED)
+# Enable fine-grained RawSpeed control
+add_definitions(-DUSE_RAWSPEED_BITS)
 
 # -- Compilation rules for RawSpeed library -------------------------------------------------------------
 
